engine/core/src/main/java/org/datacleaner/database/DatabaseDriverCatalog.java

```diff
@@ -263,4 +286,164 @@ public boolean isInstalled(String databaseName) {
         }
         return getState(databaseDriver) == DatabaseDriverState.INSTALLED_WORKING;
     }
+    
+    /**
+     * Returns the descriptors of datastore types available in DataCleaner.
+     */
+    public List<DatastoreDescriptor> getAvailableDatastoreDescriptors() {
```

Comment:
The changes/additions here in this class seems redundant with the complete DatastoreDescriptorDesktopBindings class?